tools/cbmc/proofs/TCP/prvTCPPrepareSend/README.md

```diff
@@ -0,0 +1,15 @@
+This is the memory safety proof for prvTCPPrepareSend.
+
+This proof is a work-in-progress.  Proof assumptions are described in
```

Comment:
There will for sure be more changes to these proofs in the near term? That's fine of course, but if not, might as well remove this sentence.

Reply:
We are in the middle of checking in all 40+ proofs sitting in outstanding pull requests as they are, because they are correct and have value and because we want them to keep current with code changes.

But they make assumptions, and in the next pass we will do a better job of extracting the assumptions, stating the assumptions, and making the assumptions as weak as possible (to make the proofs as strong as possible).